### ISSUE TITLE
[request] WebVR build flavor.

### ISSUE THREAD

**Issue by Codes4Fun** [nwjs/nw.js#4259]
I was wondering if someone could create a [WebVR](http://webvr.info/) build flavor:

https://chromium.googlesource.com/experimental/chromium/src/+/refs/wip/bajones/webvr

With Oculus and Valve releasing HMDs this year, it would be nice to have pre-packaged solution for VR Web Apps that can be distributed to Windows, Linux, and Macs in Valve Steam and Oculus Home.

If this isn't too strange of a request I'll put a bounty on it.

**Comment by arthurjamain** [comment]
Hey there !

Is this any traction on such issues ? We at Sketchfab very much would like an app shell that handles WebVR and may not have enough resources to develop it. The objective would precisely be what you mentioned in your original post in January :) Nice anticipation.

Anyways, we can and will pay substantially. Is it worth adding onto this bounty ? should I create another one ? Or not bother because people don't actually do that ?

Thanks !

-- Arthur, Front @ Sketchfab

**Comment by GnorTech** [comment]
We would like to support this. But it takes more efforts than the bounty to maintain a new build flavor. And currently we're in short of resources. See also https://salt.bountysource.com/teams/nwjs

**Comment by arthurjamain** [comment]
Makes sense, thanks for your answer. I'll look at alternative approaches. 
Cheers on the continued development !
